README.md

```diff
@@ -24,7 +24,8 @@ environments, develop in SNAFU and then write benchmark-operator benchmark to in
 | OpenShift Upgrade              | Time to upgrade                | Working            |
 | OpenShift Scaling              | Time to scale                  | Working            |
 | Log Generator                  | Log throughput to backend      | Working            |
-
+| DNS-Perf                       | DNS Performance                | Working            |
```

Comment:
Add documentation around the wrapper - what the workload is and the metrics captured.

Reply:
Done, added documentation around the wrapper.